### PR TITLE
Add scope validation for subscriptions and applications

### PR DESCRIPTION
**Pull request by ruwiniwj**
Add scope validation for following store UI operations
- Subscriptions - Add, Delete
- Applications - Add, Delete, Edit, Token generation

Fix issue https://github.com/wso2/carbon-apimgt/issues/6543

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Credentials/Credentials.jsx [hunk 1]
```diff
@@ -38,6 +38,7 @@ import InlineMessage from '../../../Shared/InlineMessage';
 import ExpressMode from './ExpressMode';
 import Subscription from '../../../../data/Subscription';
 import Alert from '../../../Shared/Alert';
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Credentials/Credentials.jsx]: Can you please use the webpack path alias here, i:e `AppData`

Comment (ruwiniwj) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Credentials/Credentials.jsx]: Fixed

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Credentials/Credentials.jsx [hunk 2]
```diff
@@ -80,6 +81,7 @@ const styles = theme => ({
         borderBottom: 'solid 1px ' + theme.palette.grey.A200,
         fontSize: '11px',
         paddingLeft: theme.spacing.unit,
+        height: 35,
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Credentials/Credentials.jsx]: Shall we use `theme.spacing.unit,` to define height here ?

Comment (ruwiniwj) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Credentials/Credentials.jsx]: fixed with https://github.com/wso2/carbon-apimgt/pull/6728

**Pull request merged**